### PR TITLE
cloud: Sink to TiDB Cloud

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/changefeed-sink-to-tidb-cloud.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    - **Tables to be replicated**: this column shows the tables to be replicated. But it does not show the new tables to be replicated in the future or the schemas to be fully replicated.
+    - **Tables without valid keys**: this column shows tables without unique and primary keys. For these tables, because no unique identifier can be used by the downstream system to handle duplicate events, their data might be inconsistent during replication. To avoid such issues, it is recommended that you add unique keys or primary keys to these tables before the replication, or set filter rules to filter out these tables. For example, you can filter out the table `test.tbl1` using "!test.tbl1".
+
+6. In **Start Position**, configure the starting position for your TiDB Cloud sink.
+
+    - If you have [loaded the existing data](#load-existing-data-optional) using Dumpling, select **Start replication from a specific TSO** and fill in the TSO that you get from Dumpling exported metadata files.
+
+    <!--
```
Comment (qiancai) [inline tidb-cloud/changefeed-sink-to-tidb-cloud.md:99]: TBD

File: tidb-cloud/changefeed-sink-to-tidb-cloud.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+8. Click **Next** to review the Changefeed configuration.
+
+    If you confirm all configurations are correct, check the compliance of cross-region replication, and click **Create**.
+
+    If you want to modify some configurations, click **Previous** to go back to the previous configuration page.
+
+9. The sink starts soon, and you can see the status of the sink changes from "**Creating**" to "**Running**".
+
+    Click the Changefeed name, and you can see more details about the changefeed, such as the checkpoint, replication latency, and other metrics.
```
Comment (hfxsd) [inline tidb-cloud/changefeed-sink-to-tidb-cloud.md:97]:
```suggestion
    Click the changefeed name, and you can see more details about the changefeed, such as the checkpoint, replication latency, and other metrics.
```

File: tidb-cloud/changefeed-sink-to-tidb-cloud.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    - In the **Changefeed Specification** area, specify the number of Replication Capacity Units (RCUs) to be used by the changefeed.
+    - In the **Changefeed Name** area, specify a name for the changefeed.
+
+8. Click **Next** to review the Changefeed configuration.
+
+    If you confirm all configurations are correct, check the compliance of cross-region replication, and click **Create**.
+
+    If you want to modify some configurations, click **Previous** to go back to the previous configuration page.
+
+9. The sink starts soon, and you can see the status of the sink changes from "**Creating**" to "**Running**".
```
Comment (hfxsd) [inline tidb-cloud/changefeed-sink-to-tidb-cloud.md:95]: 双引号可以去掉吧？保留就可以了。

**Review by hfxsd — APPROVED** [review]
Rest LGTM

File: tidb-cloud/changefeed-sink-to-tidb-cloud.md (hunk 3 above)
Comment (qiancai) [inline tidb-cloud/changefeed-sink-to-tidb-cloud.md:95]:
```suggestion
9. The sink starts soon, and you can see the status of the sink changes from **Creating** to **Running**.
```

File: tidb-cloud/changefeed-sink-to-tidb-cloud.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+- If the table to be replicated does not have a primary key or a non-null unique index, the absence of a unique constraint during replication could result in duplicated data being inserted downstream in some retry scenarios.
+- The **Sink to TiDB Cloud** feature is only available to TiDB Cloud Dedicated Tier clusters that are in the following AWS regions and created after November 9, 2022:
+
+    - AWS Oregon (us-west-2)
+    - AWS Frankfurt (eu-central-1)
+    - AWS Singapore (ap-southeast-1)
+    - AWS Tokyo (ap-northeast-1)
+
+- The **Sink to TiDB Cloud** feature only supports network connection via private endpoints. When you create a changefeed to stream data from a TiDB Cloud Dedicated Tier cluster to a TiDB Cloud Serverless Tier cluster, TiDB Cloud will automatically set up the private endpoint connection between the two clusters.
+
```
Comment (qiancai) [inline tidb-cloud/changefeed-sink-to-tidb-cloud.md:27]: 是否需要加一个限制，说明只支持相同 region 的 sink

**Review by Frank945946 — APPROVED** [review]

File: tidb-cloud/changefeed-sink-to-tidb-cloud.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
+---
+title: Sink to TiDB Cloud Serverless Tier clusters
```
Comment (qiancai) [inline tidb-cloud/changefeed-sink-to-tidb-cloud.md:2]:
```suggestion
title: Sink to TiDB Cloud
```

**Pull request merged**